### PR TITLE
test(e2e): add modal-navigation app tests

### PR DESCRIPTION
**Pull request by vchimev**
Tests for https://github.com/NativeScript/NativeScript/pull/5393.

### CODE REVIEW

File: e2e/modal-navigation/e2e/tests.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// import { AppiumDriver, createDriver, SearchOptions } from "nativescript-dev-appium";
```
Comment (vakrilov) [inline e2e/modal-navigation/e2e/tests.e2e-spec.ts]: Is this file needed at all? If not - delete it.

**Comment by SvetoslavTsenov** [comment]
test

**Comment by SvetoslavTsenov** [comment]
test

**Comment by SvetoslavTsenov** [comment]
test

**Comment by SvetoslavTsenov** [comment]
test branch_modal_navigation#vchimev/e2e/modal-navigation

**Comment by vchimev** [comment]
test branch_modal_navigation#vchimev/e2e/modal-navigation

**Comment by vchimev** [comment]
test branch_modal_navigation#vchimev/e2e/modal-navigation

**Comment by SvetoslavTsenov** [comment]
test branch_modal_navigation#vchimev/e2e/modal-navigation

**Comment by SvetoslavTsenov** [comment]
test branch_modal_navigation#vchimev/e2e/modal-navigation

**Comment by vchimev** [comment]
test branch_modal_navigation#vchimev/e2e/modal-navigation

**Comment by vchimev** [comment]
test --ignore branch_modal_navigation#vchimev/e2e/modal-navigation

**Comment by vchimev** [comment]
test branch_modal_navigation#vchimev/e2e/modal-navigation
